### ISSUE TITLE
Fix impacts due to ruleID changes when getCache(ruleid) is used

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1273]
ruleIDs also show up in `getCache(ruleid)` in the code about 54 times (in ~54 rules) and needs to be handled when the names are changed. In most cases the order of the rule execution affects the ability and option to use the cache
- cache was created because the rule was run before the current rule and cached the results for later rules to utilize

There are various scenarios or approaches that will be needed to resolve ruleID changes:
- [x] NA ~~create new functional utilities to replace the cache approach.~~
- [x] investigate that the remaining simple cache approach is still valid when/because the rule is run prior because it retains the correct "alphabetical" execution order.

This impact will need to be assessed and sized in this work issue.

**Comment by sct-will** [comment]
Dependent on another issue in progress.

**Comment by philljenkins** [comment]
Dependent on #1271 currently in progress

**Comment by MHoov** [comment]
@philljenkins #1271 is closed

**Comment by sct-will** [comment]
Need further discussion to move forward on this one.

**Comment by philljenkins** [comment]
Picking this back up after focusing on ACT rules

**Comment by philljenkins** [comment]
A review meeting held with Tom, Shunguo, and me will close this issue.
Opening a new issue to do the fixes mentioned above.
